### ISSUE TITLE
SendPingAsyncWithHostAndTimeoutAndBuffer_Unix failed in CI

### ISSUE THREAD

**Issue by stephentoub** [dotnet/corefx#37370]
```
System.Net.NetworkInformation.Tests.PingTest.SendPingAsyncWithHostAndTimeoutAndBuffer_Unix


Error message
Assert.Equal() Failure\nExpected: InterNetworkV6\nActual: InterNetwork

Stack trace
   at System.Net.NetworkInformation.Tests.PingTest.PingResultValidator(PingReply pingReply, IPAddress[] localIpAddresses) in /Users/vsts/agent/2.150.3/work/1/s/src/System.Net.Ping/tests/FunctionalTests/PingTest.cs:line 58
   at System.Net.NetworkInformation.Tests.PingTest.<>c__DisplayClass27_0.<SendPingAsyncWithHostAndTimeoutAndBuffer_Unix>b__1(PingReply pingReply) in /Users/vsts/agent/2.150.3/work/1/s/src/System.Net.Ping/tests/FunctionalTests/PingTest.cs:line 530
   at System.Net.NetworkInformation.Tests.PingTest.SendPingAsync(Func`2 sendPing, Action`1 pingResultValidator) in /Users/vsts/agent/2.150.3/work/1/s/src/System.Net.Ping/tests/FunctionalTests/PingTest.cs:line 778
   at System.Net.NetworkInformation.Tests.PingTest.SendPingAsyncWithHostAndTimeoutAndBuffer_Unix() in /Users/vsts/agent/2.150.3/work/1/s/src/System.Net.Ping/tests/FunctionalTests/PingTest.cs:line 526
--- End of stack trace from previous location where exception was thrown ---
```
cc: @wfurt

**Comment by wfurt** [comment]
Do you have link to the failure @stephentoub ? I assume this is OSX?

**Comment by stephentoub** [comment]
Yup:
https://mc.dot.net/#/user/dotnet-bot/pr~2Fdotnet~2Fcorefx~2Frefs~2Fpull~2F37368~2Fmerge/test~2Ffunctional~2Fcli~2F/20190502.14/workItem/System.Net.Ping.Functional.Tests/analysis/xunit/System.Net.NetworkInformation.Tests.PingTest~2FSendPingAsyncWithHostAndTimeoutAndBuffer_Unix

**Comment by wfurt** [comment]
Seems related to  #37362.  The assert is confusing but it is on path where ping timed out.

**Comment by wfurt** [comment]
this should be fixed by #37284 submitted on Apr 30. 
All the info is now gone so it hard to know if that was part if the failing PR. I did check Kusto database and there is no other failure so I'm gonna close it for now.
Note that there are other failures with timeout, probably related to overall CI and name lookup stability.
